album/server.py

```diff
@@ -25,14 +39,59 @@ def run_index():
 
 
 @app.route('/run/<uid>')
-def run_show(uid):
+def run_show(*args, **kwargs):
```

Comment:
long story short I was trying to debug bokeh oddities. This needs to be reverted